docs/source/config.json

```diff
@@ -111,6 +112,15 @@
       },
       true
     ],
+    "Pagination": [
+      {
+        "Introduction": "/pagination/introduction",
+        "Using Custom Response Models": "/pagination/custom-types",
+        "Forward, Reverse, and Bi-directional Pagination": "/pagination/directional-pagers",
```

Comment:
This is going to look really long in the side navigation bar.

Since forward pagination is pretty straightforward and explained in the intro page, I think maybe we could change this to "Reverse Pagination" or "Directional Pagination"? Not sure what makes the most sense.